types/index.d.ts

```diff
@@ -0,0 +1,10 @@
+interface Cordova {
+    getAppVersion: GetAppVersion;
+}
+
+interface GetAppVersion {
+    getAppName(): Promise<string>;
+    getPackageName(): Promise<string>;
+    getVersionNumber(): Promise<string>;
+    getVersionCode(): Promise<string>;
```

Comment:
Just double-checking - `getPromisedCordovaExec` can return a number of different types of Promise (`jQuery.Deferred()`, an angular deferred promise, `when.defer().promise`, a `Promise` object, or a `WinJS.Promise` object). Can all of these be considered the same `Promise` type? Do you risk losing type information that's specific to JQuery, Angular of WinJS by doing this?

Context: I'm hosting this code as the org it was originally hosted under is going away, but I didn't write it so don't know much about it!

Reply:
I'm not sure. I've only used vanilla Promises

Reply:
No problem. If ever you do get a chance to work this out and tweak the type definitions as appropriate, then I'll merge this PR. Otherwise I'm happy to leave it open so that others are aware of your fork or for others to build off. Thanks again for your involvement.